front-end-master/about_us/about.js

```diff
@@ -58,16 +9,106 @@ function divSelectorClassRemover(hoveredDivIndex, divIndexToRemove=4){
 }
 
 function outClickListener(){
-	console.log(event.target);
 	if(!$(event.target).closest('#pop-up-window').length) {
-		console.log('called');
 		if($('.pop-up-container')[0].classList.contains('pop-up-shown')) {
 			$('.pop-up-container')[0].classList.remove('pop-up-shown');
 			$('.pop-up-container')[0].classList.add('pop-up-hidden');
 		}
 	}
 }
 
+//<div id="pop-up-window" class="pop-up">
+//	<div class="profile-data">
+//		<img src="members/intekhab_hossain.jpg">
+//		<div class="info">
+//			<span class="name"></span>
+//			<span class="designation"></span>
+//			<span class="project-name"></span>
+//		</div>
+//		<div class="social-links">
+//			<a href="#"><img src="icons/email.svg"></a>
+//			<a href="#"><img src="icons/facebook.svg"></a>
+//			<a href="#"><img src="icons/instagram.svg"></a>
+//			<a href="#"><img src="icons/twitter.svg"></a>
+//		</div>
+//		<img src="icons/cross.svg" class="cross">
+//	</div>
+//
+//	<div class="bio">
+//		<div class="info">
+//			<div class="line">
+//				<span class="title"></span>
+//				<p></p>
+//			</div>
+//			<div class="line">
+//				<span class="title"></span>
+//				<p></p>
+//			</div>
+//			<div class="line">
+//				<span class="title"></span>
+//				<p></p>
+//			</div>
+//		</div>
+//
+//		<div class="main-bio">
+//			<p> </p>
+//		</div>
+//	</div>
+//</div>
+
+
+function getChild(elem, className){
+	var childs = [];
+	for (var i = 0; i < elem.childNodes.length; i++) {
+		if (elem.childNodes[i].className == className) {
+		  childs.push(elem.childNodes[i]);
+		}
+	}
+	return childs;
+}
+
+function populatePop_Up(popUp, person){
+	let profile_data = getChild(popUp, 'profile-data')[0];
```

Comment:
Can we refactor this out into functions like: populateBio(), populateProfile(), ?

Reply:
explain? you want me to make two functions to populate two parts of the popUp? didn't understand that

Reply:
on a second thought, it will make sense to have the properties related to a profile, i.e. name, designation,project_name encapsulated into a TeamMember object. I know you had a Person object. Where are we using that?